src/cmd/common/table.go

```diff
@@ -14,6 +14,27 @@ import (
 	"github.com/zarf-dev/zarf/src/pkg/message"
 )
 
+const (
+	COMPONENTS string = "components:"
+	NAME       string = "  - name: %s\n"
+	IMAGES     string = "    images:"
+	IMAGE      string = "      - %s\n"
+)
+
+// PrintComponentTable prints the components with found images.
+func PrintComponentTable(imagesMap map[string][]string) {
+	fmt.Println(COMPONENTS)
+	for component, images := range imagesMap {
+		fmt.Printf(NAME, component)
+		if len(images) > 0 {
+			fmt.Println(IMAGES)
+			for i := 0; i < len(images); i++ {
+				fmt.Printf(IMAGE, images[i])
```

Comment:
I was not sure if printing them out with the `fmt.Print` functions was the correct way to handle this, I did not see a good log level to put it in the logger.

Please let me know if you want me to update this